homeassistant/components/netatmo/climate.py

```diff
@@ -109,19 +109,24 @@ def __init__(self, data, room_id):
         self._state = None
         self._room_id = room_id
         room_name = self._data.homedata.rooms[self._data.home][room_id]['name']
-        self._name = 'netatmo_' + room_name
+        self._name = "netatmo_{}".format(room_name)
```

Comment:
Why switch from single to double quotes?

Reply:
Changed back to single quote.